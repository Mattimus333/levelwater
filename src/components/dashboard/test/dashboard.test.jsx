import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Dashboard from '../dashboard.jsx';

describe('dashboard', () => {
  it('renders a snapshot', () => {
    const tree = shallow(<Dashboard />);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
