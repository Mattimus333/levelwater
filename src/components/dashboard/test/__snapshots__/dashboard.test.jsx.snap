exports[`dashboard renders a snapshot 1`] = `
<div
  id="dashboard">
  <DashHeader />
  <C3Visuals />
</div>
`;
